libraries/AP_HAL_ChibiOS/hwdef/StellarH7V2/Readme.md

```diff
@@ -0,0 +1,114 @@
+# StellarH7V2 Flight Controller
```

Comment:
Needs to be two commits, one for each subsystem. The commit message should start with:
"bootloaders: " - for the bootloader changes and
"AP_HAL_ChibiOS: " - for the hwdef changes